### ISSUE TITLE
Skarsnik's Waagh ability uses an Orc soundbite instead of a Goblin soundbite. 

### ISSUE THREAD

**Issue by floskan** [chadvandy/cbfm_wh3#2011]
https://community.creative-assembly.com/total-war/total-war-warhammer/bugs/5268-skarsnik-s-waagh-ability-uses-an-orc-soundbite-instead-of-a-goblin-soundbite?page=1

**Comment by mfvreeland** [comment]
Is there a Goblin soundbyte for this? Has this ever worked differently? Needs confirmation.

**Comment by floskan** [comment]
> confirmation

Yeah there was goblins shouting when he called a waaagh previously
